index.html

```diff
@@ -0,0 +1,316 @@
+              </div>
+             <div>
+                <img class="line2" src="image/line22.png" alt="line2">
+              </div>
+          </div>
+            <div class="language">
+                <div class="lan">
+                    <img src="image/Rectangle55.png" alt="language">
+                    <h3> Language </h3>
+                </div>
+                
+                <ul>
+                    <li> JavaScript</li>
+                    <li>Ruby </li>
+                    <li>HTML </li>
+                    <li>CSS </li>
+                </ul>
+            </div>
+    
+    
+            <div class="framwork">
+                <div class="frm">
+                    <img src="image/Rectangle56.png" alt="framework">
+                    <h3> Frameworks </h3>
+                </div>
+                <ul>
+                    <li> Bootstrap</li>
+                    <li>Ruby on Rails </li>
+                    <li>RSPec </li>
+                    <li>CapyBara </li>
+                    <li>Selenium </li>
+                </ul>
+            </div>
+    
+    
+            <div class="Skills">
+                <h3> <img src="image/Rectangle57.png" alt="skill"> Skills </h3>
+                <ul>
+                    <li> Codekit </li>
+                    <li>Github </li>
+                    <li>Codepen </li>
+                    <li>Gitlab </li>
+                    <li>Terminal </li>
+                </ul>
+            </div>
+    
+        </div>
+    </section>
+    <div class="form-page">
+        <div class="p-form">
+            <p>
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+        </div>
+        <div class="forms0">
+            <form action="https://formspree.io/f/xoqrrprd" method="post">
+                <ul class="forms1">
+                    <li>
+                        <input type="text" id="fullname" name="fname" maxlength="30" required placeholder="Your full name">
+                    </li>
+                    <li>
+                        <img class="form-line" src="form-image/line111.png" alt="line">
+                    </li>
+                    <li>
+                        <input type="email" id="emaail" name="mail" required placeholder="Email adress"
+                            title="please enter valid email address like me@something.com">
+                    </li>
+                    <li>
+                        <img class="form-line" src="form-image/line111.png" alt="line">
+                    </li>
+                    <li>
+                        <textarea name="message" rows="4" cols="30" maxlength="500" required
+                            placeholder="Enter text here"></textarea>
+                    </li>
+                </ul>
+    
+                <div class="get-in-touch">
+                    <input type="submit" value="Get in touch">
+                </div>
```

Comment:
According to the project requirements, you should implement button interactions.

<img width="909" alt="Screen Shot 2022-05-30 at 18 38 01" src="https://user-images.githubusercontent.com/82426682/171068357-51818f78-047e-48ad-b912-1ac56229ffe9.png">

Your buttons don't have any interactions, kindly fix that.